feature-detail/src/androidMain/kotlin/detail/di/DetailStreamModule.kt

```diff
@@ -1,14 +1,14 @@
-package com.codandotv.streamplayerapp.feature_list_streams.detail.di
+package detail.di
 
-import com.codandotv.streamplayerapp.feature_list_streams.detail.data.DetailStreamRepository
-import com.codandotv.streamplayerapp.feature_list_streams.detail.data.DetailStreamRepositoryImpl
-import com.codandotv.streamplayerapp.feature_list_streams.detail.data.DetailStreamService
-import com.codandotv.streamplayerapp.feature_list_streams.detail.data.DetailStreamServiceImpl
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.DetailStreamUseCase
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.DetailStreamUseCaseImpl
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.VideoStreamsUseCase
-import com.codandotv.streamplayerapp.feature_list_streams.detail.domain.VideoStreamsUseCaseImpl
-import com.codandotv.streamplayerapp.feature_list_streams.detail.presentation.screens.DetailStreamViewModel
+import detail.data.DetailStreamRepository
```

Comment:
The same point I said before, I think it is something going wrong with our name space for this module